src/python/dxpy/templating/templates/nextflow/src/nextflow.sh

```diff
@@ -467,26 +430,7 @@ validate_run_opts() {
   done
 }
 
-dx_path() {
```

Comment:
Not used; deleting per @YuxinShi0423 's comment on previous refactoring PR

Reply:
Thanks!